pkg/cmd/packaging.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2019 The KubeOne Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/cobra/doc"
+)
+
+func completionCmd(rootCmd *cobra.Command) *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:   "completion <bash|zsh>",
+		Short: "Generates completion scripts for bash and zsh",
+		Long: `To load completion run into your current shell run
+
+. <(kubeone bashCompletion <shell>)
```

Comment:
There's a typo in the description:

```suggestion
. <(kubeone completion <shell>)
```